### PR TITLE
Cumulative sum loop

### PR DESCRIPTION
**Pull request by EvaTomlinson**

### CODE REVIEW

File: A170_E01_Tomlinson_Eva/src/CumulativeSumLoop.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+class CumulativeSumLoop {
+  public static void main(String[] args){
+    cumlSum();
```
Comment (kurttomlinson) [inline A170_E01_Tomlinson_Eva/src/CumulativeSumLoop.java]: Please give this a more descriptive name.

```suggestion
    sumUserInputs();
```

**Review by kurttomlinson — CHANGES_REQUESTED** [review]
Looks good! Just one small change requested :)

File: A170_E01_Tomlinson_Eva/src/CumulativeSumLoop.java (hunk 1 above)
Comment (EvaTomlinson) [reply A170_E01_Tomlinson_Eva/src/CumulativeSumLoop.java]: Good idea! I changed the name!

**Review by kurttomlinson — APPROVED** [review]
👍

**Pull request merged**